jobs/config.json

```diff
@@ -3233,6 +3233,30 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-coredns": {
```

Comment:
Could you instead name this as 'gce-coredns-performance' or sth to not have prefix clash with our regular scalability jobs?